src/rev_ai/models/asynchronous/summary.py

```diff
@@ -0,0 +1,20 @@
+from typing import List, Dict
+
+"""Transcript summary model."""
```

Comment:
this [should be](https://peps.python.org/pep-0008/#:~:text=Imports%20are%20always%20put%20at,before%20module%20globals%20and%20constants.) either before imports, or inside the class

```
Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
```


so the file might look like this:
```
"""Module description here"""

import xyz

class MyClass:
    """This is my class"""
    pass

class MyClass2:
    """This is my class 2"""
    pass
```